packages/theme-addresses/README.md

```diff
@@ -0,0 +1,133 @@
+
+const billingCountryProvinceSelector = new CountryProvinceSelector(window.theme.allCountryOptionTags);
+```
+
+---
+
+### build(countryNodeElement, provinceNodeElement, options)
+
+Populates the given `<select>` elements with country and province options.
+
+- `countryNodeElement`: The `<select>` node element for countries
+- `provinceNodeElement`: The `<select>` node element for provinces according to the selected country
+- `options`:
+    * `onCountryChange`: The callback after a country `change` event
+    * `onProvinceChange`: The callback after a province `change` event
+
+The Select node element should look like the following:
+```html
+    <select ... id="newCountry" data-default="{{ form.country }}"></select>
+    <select ... id="newProvince" data-default="{{ form.province }}"></select>
+```
+
+Example
+```js
+import {CountryProvinceSelector} from '@shopify/theme-addresses';
+
+const billingCountryProvinceSelector = new CountryProvinceSelector(window.theme.allCountryOptionTags);
+
+billingCountryProvinceSelector.build(document.querySelector('#newCountry'), document.querySelector('#newProvince'));
+billingCountryProvinceSelector.build(document.querySelector('#editCountry'), document.querySelector('#editProvince'));
+```
+
+We do not dictate what selector library you can use in your themes as long as the element being passed in is a native Node element.
+
+Example using jQuery
+```js
+import $ from 'jquery';
+import {CountryProvinceSelector} from '@shopify/theme-addresses';
+
+const billingCountryProvinceSelector = new CountryProvinceSelector(window.theme.allCountryOptionTags);
+
+billingCountryProvinceSelector.build($('#newCountry')[0], $('#newProvince')[0]);
+billingCountryProvinceSelector.build($('#editCountry')[0], $('#editProvince')[0]);
+```
+
+### onCountryChange(provinces, provinceNodeElement, countryNodeElement)
+
+The callback when a `change` event happens on the country select.
+
+- `provinces`: The array of provinces of selected country
+- `provinceNodeElement`: The `<select>` node element for provinces
+- `countryNodeElement`: The `<select>` node element for countries
+
+### onProvinceChange(event)
+
+The callback when a `change` event happens on the province select.
+
+- `event`: The `change` event object of the province select
+
+Example
+
+```html
+    <select ... id="newCountry" data-default="{{ form.country }}"></select>
+    <div id="newProvinceContainer">
+        <p>Provinces</p>
+        <select ... id="newProvince" data-default="{{ form.province }}"></select>
+    </div>
+```
+
+```js
+import {CountryProvinceSelector} from '@shopify/theme-addresses';
+
+const newProvinceContainer = document.querySelector('#newProvinceContainer')
+const billingCountryProvinceSelector = new CountryProvinceSelector(window.theme.allCountryOptionTags);
+
+billingCountryProvinceSelector.build(
+    document.querySelector('#newCountry'),
+    document.querySelector('#newProvince'),
+    onCountryChange: (provinces) => {
```

Comment:
You're missing an object `{ ... }` wrapper for this key:value. 

Also the ways the docs read, it made it look like `onProvinceChange()` is a method of CountryProvinceSelector, e.g:
```
billingCountryProvinceSelector.onCountryChange = (provinces) => { ... };
```

Reply:
🦅 👁

Reply:
I do agree about the doc style .. but I am not sure how I can make it not